packages/gmap-vue/examples/heatmap-layer.html

```diff
@@ -0,0 +1,39 @@
+<body>
+  <div id="root">
```

Comment:
```suggestion
  <div id="root">
      <button @click="setMarkers">
        Set markers on heatmap
      </button>
      <br><br>
```
Add a button to load the markers and show the heat map to the users.